Version 0.15.0 is not deployed to https://plugins.gradle.org/m2/

https://plugins.gradle.org/m2/gradle/plugin/nl/javadude/gradle/plugins/license-gradle-plugin/
does not contain version 0.15.0.

Because of this, the build in "old gradle" form fails with:

> FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring root project 'myProject'.
> Could not resolve all files for configuration ':classpath'.
   > Could not find gradle.plugin.nl.javadude.gradle.plugins:license-gradle-plugin:0.15.0.
     Searched in the following locations:
         https://jcenter.bintray.com/gradle/plugin/nl/javadude/gradle/plugins/license-gradle-plugin/0.15.0/license-gradle-plugin-0.15.0.pom
         https://jcenter.bintray.com/gradle/plugin/nl/javadude/gradle/plugins/license-gradle-plugin/0.15.0/license-gradle-plugin-0.15.0.jar
         https://repo.maven.apache.org/maven2/gradle/plugin/nl/javadude/gradle/plugins/license-gradle-plugin/0.15.0/license-gradle-plugin-0.15.0.pom
         https://repo.maven.apache.org/maven2/gradle/plugin/nl/javadude/gradle/plugins/license-gradle-plugin/0.15.0/license-gradle-plugin-0.15.0.jar
         https://plugins.gradle.org/m2/gradle/plugin/nl/javadude/gradle/plugins/license-gradle-plugin/0.15.0/license-gradle-plugin-0.15.0.pom
         https://plugins.gradle.org/m2/gradle/plugin/nl/javadude/gradle/plugins/license-gradle-plugin/0.15.0/license-gradle-plugin-0.15.0.jar
     Required by:
         project :

--- a/README.adoc
+++ b/README.adoc
@@ -34,7 +34,7 @@ buildscript {
     }
   }
   dependencies {
-    classpath "gradle.plugin.nl.javadude.gradle.plugins:license-gradle-plugin:{license_plugin_version}"
+    classpath "gradle.plugin.com.hierynomus.gradle.plugins:license-gradle-plugin:{license_plugin_version}"
   }
 }
 

